contents/blog/best-hotjar-alternatives.md

```diff
@@ -0,0 +1,431 @@
+  <ComparisonRow column1={true} column2={true} feature="Heatmaps" description="See where users click and interact" />
+  <ComparisonRow column1={true} column2={true} feature="Rage clicks" description="Detect when users click rapidly in the same place" />
+  <ComparisonRow column1={false} column2={true} feature="Scroll maps" description="See how far users scroll on websites and apps" />
+  <ComparisonRow column1={true} column2={false} feature="Product analytics" description="Custom trends, funnels, paths, and retention analysis" />
+  <ComparisonRow column1={true} column2={false} feature="Feature flags" description="Control feature access and phase rollouts" />  
+  <ComparisonRow column1={true} column2={false} feature="A/B testing" description="Test changes and analyze their impact" />
+  <ComparisonRow column1={true} column2={true} feature="Surveys" description="Collect feedback and run satisfaction surveys" />
+</ComparisonTable>
+
+*iOS replays are under development.
+
+### Why do companies use PostHog?
+
+According to [reviews on G2](https://www.g2.com/products/posthog/reviews), companies use PostHog because:
+
+1. **It replaces multiple tools:** PostHog can replace [Hotjar](/blog/posthog-vs-hotjar) (session replay & surveys), [Google Analytics](/blog/posthog-vs-ga4), and [LaunchDarkly](/blog/posthog-vs-launchdarkly) (feature flags and A/B testing). This simplifies workflows and ensures all their data is in one place.
+
+2. **Pricing is transparent and scalable:** Reviewers appreciate how PostHog's pricing scales as they grow. There's a [generous free tier](/pricing). Companies eligible for [PostHog for Startups](/startups) also get an additional $50k in credits.
+
+3. **They need a complete picture of users:** PostHog includes every tool necessary to understand users and improve products. This means creating funnels to track conversion, watching replays to see where users get stuck, testing solutions with A/B tests, and gathering feedback with user surveys.
+
+> #### Bottom line
+>
+> PostHog is a broader, more powerful tool than Hotjar. This comes with some extra complexity, but the payoff is all your user data in one place and tightly integrated with powerful analytical tools.
+
+<ArrayCTA />
+
+<br />
+
+## 2. Mouseflow
+
+- **Founded:** 2009
+- **Most similar to:** Hotjar
+- **Typical users:** Marketing and UX teams
+- **Typical customers:** SMB and enterprises in retail
+
+![mouseflow](../images/blog/clarity-alternatives/mouseflow.png)
+
+### What is Mouseflow?
+
+Mouseflow is a behavioral analytics tool that offers session replays, heatmaps, and basic funnel analysis. It's designed mainly for marketing and e-commerce websites. It's very similar to Hotjar, making it a direct rip and replace solution.
+
+According to data from BuiltWith, 5,555 of the top 1 million websites use Mouseflow (correct as of Feb 2024), considerably less than Hotjar's 72,048.
```

Comment:
Similar comment re above. Not sure how helpful this is to have thie BuiltWith data and just feels like extra text to read. Perhaps we should remove from all the sections